internal/clients/sources/sources_payloads.go

```diff
@@ -0,0 +1,60 @@
+package sources
+
+type AuthResponse struct {
+	Authtype     string `json:"authtype"`
+	ID           string `json:"id"`
+	ResourceID   string `json:"resource_id"`
+	ResourceType string `json:"resource_type"`
+	Username     string `json:"username"`
+}
+
+type CreateSourceResponse struct {
+	Applications []struct {
+		ApplicationTypeID string `json:"application_type_id"`
+		CreatedAt         string `json:"created_at"`
+		ID                string `json:"id"`
+		SourceID          string `json:"source_id"`
+		UpdatedAt         string `json:"updated_at"`
+	} `json:"applications"`
+	Authentications []struct {
+		Authtype     string `json:"authtype"`
+		ID           string `json:"id"`
+		ResourceID   string `json:"resource_id"`
+		ResourceType string `json:"resource_type"`
+		Username     string `json:"username"`
+	} `json:"authentications"`
+	Endpoints []interface{} `json:"endpoints"`
+	Sources   []struct {
+		AppCreationWorkflow string `json:"app_creation_workflow"`
+		CreatedAt           string `json:"created_at"`
+		ID                  string `json:"id"`
+		Name                string `json:"name"`
+		SourceTypeID        string `json:"source_type_id"`
+		UID                 string `json:"uid"`
+		UpdatedAt           string `json:"updated_at"`
+	} `json:"sources"`
+}
+
+type BulkCreateRequest struct {
+	Sources         []Source         `json:"sources"`
+	Applications    []Application    `json:"applications"`
+	Authentications []Authentication `json:"authentications"`
+}
+
+type Source struct {
+	Name                string `json:"name"`
+	SourceTypeName      string `json:"source_type_name"`
+	AppCreationWorkflow string `json:"app_creation_workflow"`
+}
+
+type Application struct {
+	SourceName          string `json:"source_name"`
+	ApplicationTypeName string `json:"application_type_name"`
+}
+
+type Authentication struct {
+	ResourceType string `json:"resource_type"`
+	ResourceName string `json:"resource_name"`
+	Username     string `json:"username"`
+	Authtype     string `json:"authtype"`
+}
```

Comment:
Is this generated? If so, can you provide information on how you generated this?